data/coalition/coalition missions.txt

```diff
@@ -3211,3 +3211,585 @@ mission "Heliarchs Buy Jump Drive Mission"
+			label go
+			`	Anna heads to your ship, and you plot a course to <destination>. Just before you launch, the Heliarch hops in.`
+				launch
+	on complete
+		event "beyond the hyperdrive is published" 165 197
+		conversation
+			`Landing on <planet>, Anna is excitedly holding a copy of 'Beyond the Hyperdrive.' She hastily opens the hatch as soon as it unlocks.`
+			`	Anna stops, waiting for you to catch up. "In the message they sent me, they included instructions on how to reach their offices," she explains. "Let's go!"`
+			`	You begin following Anna to the publisher's offices. You walk through the spaceport and its endless rows of shops and streets, eventually approaching what looks like a train station. Anna looks at the message again and tells you which platform to go to and which train to take. After a few minutes the train arrives. You and Anna board the train and travel to a neighborhood, with a large white building dominating the skyline.`
+			`	Once the train stops, you get off onto the platform and out of the station. You walk around the building and find an entrance, busy with many Kimek and the occasional Saryd and Arach entering and exiting. Anna looks for a front desk and directs you to walk to a Kimek who looks like a receptionist.`
+			choice
+				`	"Hello, do you happen to know where Priyepichia Publishing is?"`
+				`	"We're looking for a publishing company called Priyepichia Publishing. Could you provide directions?"`
+			`	"On the 16th floor, Priyepichia Publishing is. Over there, the elevators are," says the receptionist's translator.`
+			`	You turn to where the Kimek points you and see six large elevators, larger than what you are familiar with in human space. You look at all six and choose to enter the least crowded one.`
+			`	You and Anna enter the elevator. You hear the noises of the Kimek communicating in their native language and get the feeling they're talking about you.`
+			`	The elevator makes many stops at each floor of the building, as more and more Kimek reach their destinations. When the door opens on the 16th floor, the elevator is now half-full; and before you is a Kimek wearing an ornate jeweled necklace as well as a translation device.`
+			`	"Greetings, humans. Follow me, you must, so discuss your book, we can," the Kimek says through its translation box.`
+			`	At the direction of the Kimek, you and Anna walk as directed until you arrive at a room with glass walls. Along the way, you pass a lounge, a conference area where multiple Kimek are discussing something at a table with a group of Heliarchs, and a large open workspace where individual Kimek sit at their own computers.`
+			`	After a few minutes of walking, there is an office where another Kimek and a team of five Heliarchs are sitting. The Kimek directing you points you towards that office and walks over to speak with the group.`
```

Comment:
Describing the Heliarch species the first time they're seen, and more thematic with equal Heliarch representation from each species
```suggestion
			`	After a few minutes of walking, there is an office where another Kimek sits with a team of six Heliarchs - two from each species. The Kimek guiding you gestures toward the office, and heads over to speak with the group.`
```